elements/pl-matrix-component-input/pl-matrix-component-input.mustache

```diff
@@ -46,7 +46,7 @@
 </script>
 <span id="pl-matrix-component-input-submission-{{uuid}}">
     {{#label}}<span>{{label}}</span>{{/label}}
-    <a role="button" class="btn btn-light btn-sm" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" title="Error" tabindex="0" data-content="{{parse_error}}{{#raw_submitted_answer}}<hr><h5><small class='text-uppercase'>What you submitted</small></h5><pre>{{raw_submitted_answer}}</pre>{{/raw_submitted_answer}}">INVALID
+    <a role="button" class="btn btn-light btn-sm" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" title="Error" tabindex="0" data-content="{{parse_error}}{{#raw_submitted_answer}}<hr><h5><small class='text-uppercase'>What you submitted</small></h5>{{raw_submitted_answer}}{{/raw_submitted_answer}}">INVALID
```

Comment:
It looks like some of the popovers display the `raw_submitted_answers` while others don't? Should we standardize?